rules/cross-platform/initial_access_zoom_meeting_with_no_passcode.toml

```
@@ -37,8 +38,8 @@ tags = [
 type = "query"
 
 query = '''
-event.type:creation and event.module:zoom and event.dataset:zoom.webhook and
-  event.action:meeting.created and not zoom.meeting.password:*
+event.type:creation and event.module:zoom and event.dataset:zoom.webhook
```

```suggestion
event.type:creation and event.dataset:zoom.webhook
```

Now that we have a Fleet integration for Zoom, I think we need to remove the `event.module` value from this rule and add `logs-zoom*` to the list of indexes: `index = ["filebeat-*", "logs-zoom*"]`